.gitignore

```diff
@@ -24,6 +24,7 @@ tmtags
 *.un~
 Session.vim
 *.swp
+.vscode/*
```

Comment:
I think we should keep the VS Code `settings.json` in the repository. Hopefully it will help developers onboard quicker, by enabling things like ESLint and PHPSniffer on save. What do you think?

Reply:
I feel like all settings should be in .gitignore. For me, it was causing problems with the linter. Other people's setups might be different, no?

Reply:
Good points, this should be solved by #12